docs/site/content/docs/assets/cli-install-windows.md

```diff
@@ -1,19 +1,34 @@
 ## Installation Procedure
 
-1. Make sure you have the [Chocolatey package manager installed](https://chocolatey.org/install).
+You can install Tanzu Community Edition using Chocolatey **or** you can download the binary and install.
+### Option 1: Use the Chocolatey installation method
 
-1. You must download and install the latest version of `kubectl`. For more information, see [Install and Set Up kubectl on Windows](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/) in the Kubernetes documentation.
+1. Make sure you have the [Chocolatey](https://chocolatey.org/install) package manager installed.
 
-1. You must download and install the latest version of `docker`. For more information, see [Install Docker Desktop on Windows](https://docs.docker.com/desktop/windows/install/) in the Docker documentation.
-
-1. Open PowerShell **as an administrator** and run the following:
+1. Open Windows PowerShell **as an administrator** and run the following:
 
     ```sh
     choco install tanzu-community-edition
     ```
 
-    > Both `docker` and `kubectl` are required to be present on the system, but are not explicit Chocolatey dependencies.
-    > Installing the Tanzu Community Edition package will extract the binaries and configure the plugin repositories. This might take a minute.
-
 1. The `tanzu` command will be added to your `$PATH` variable automatically by Chocolatey.
 
+### Option 2: Use the Binary download/installation method
+
+1. Download the release for [Windows](https://github.com/vmware-tanzu/community-edition/releases/download/{{< release_latest >}}/tce-windows-amd64-{{< release_latest >}}.zip).
+
+1. Open Windows PowerShell **as an administrator**, change to the download directory and unpack the release, for example,
+
+    ```sh
+    cd <DOWNLOAD-DIR>
+    Expand-Archive -Path 'tce-windows-amd64-{{< release_latest >}}.zip'
+    ```
+
+1. Change to the extracted directory and run `install.bat`.
+
+    ```sh
+    cd tce-windows-amd64-{{< release_latest >}}
```

Comment:
The path is actually

cd tce-windows-amd64-{{< release_latest >}}\tce-windows-amd64-{{< release_latest >}}

Reply:
agree